lmos-classifier-workbench-demo-controller/src/main/kotlin/org/eclipse/lmos/classifier/api/DocumentController.kt

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: 2025 Deutsche Telekom AG and others
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.eclipse.lmos.classifier.api
+
+import org.eclipse.lmos.classifier.core.SystemContext
+import org.eclipse.lmos.classifier.embeddings.DocumentHandler
+import org.springframework.http.MediaType
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api/v1/embeddings", produces = [MediaType.APPLICATION_JSON_VALUE])
+class DocumentController(
+    private val documentHandler: DocumentHandler,
+) {
+    @PostMapping
+    fun ingestDocuments(
+        @RequestParam(defaultValue = "europe") tenant: String,
```

Comment:
Should we remove the default values, because they might not apply to everyone.

Reply:
Removed.